store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9609,4 +9609,9 @@ TODO: delete them permanently from here
   <desc>Information about the latest run of zmmigrateattrs. Includes the URL of the destination ephemeral store and the state of the migration (in progress, completed, failed)</desc>
 </attr>
 
+<attr id="3020" name="zimbraReverseProxyStrictServerName" type="boolean" cardinality="single" optionalIn="globalConfig,server" flags="serverInherited" since="8.8.1" requiresRestart="nginxproxy">
+  <globalConfigValue>FALSE</globalConfigValue>
+  <desc>Controls if the default server block in 'nginx.conf.web.https.default.template' is includedq returning a '400' status to all unknown host names.</desc>
```

Comment:
I could make an additional attribute that becomes the actual return code. 
What do you think of that?

Reply:
I will fix the typo and update the wording.

Reply:
I'd prefer to avoid creating yet another attribute/config var at this point, but others are welcome to chime in.

Reply:
I could remove the status code portion and just specify that it will enable the nginx configuration block that handles all unknown hostnames and returns a specified response.